src/Http/Controllers/MeetupController.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Controllers;
+
+use Blumilk\Meetup\Core\Models\Meetup;
+use Illuminate\Http\Request;
+
+class MeetupController extends Controller
+{
+    public function __construct()
+    {
+        $this->middleware(["auth"])->except("index");
+    }
+
+    public function index()
+    {
+        $meetups = Meetup::latest()->with(["user"])->paginate(20);
```

Comment:
It is not better to transfer such things to the site that the controller is small ?

Reply:
Its small query. That has not a big logic or smth like that. I would stay with that solution.

Reply:
But what when there will be a big query for more business logic? It's worth doing the division at the beginning to have order later

Reply:
For me, that complicates the project at first, but **ok** as you like.